.agignore

```diff
@@ -0,0 +1 @@
+target/*
```

Comment:
I'm not too picky about dot files (projectile, idea, jshint, etc) making it in. Will take PR to expunge and gitignore if it matters to you :skull